pkg/sql/execinfrapb/processors_sql.proto

```diff
@@ -121,6 +121,12 @@ message TableReaderSpec {
   // to BLOCK when locking_strength is FOR_NONE.
   optional sqlbase.ScanLockingWaitPolicy locking_wait_policy = 11 [(gogoproto.nullable) = false];
 
+  // Indicates that misplanned ranges metadata should not be sent back to the
+  // DistSQLReceiver. This will be set to true for the scan with a hard limit
+  // (in which case we create a single processor that is placed at the
+  // leaseholder of the beginning of the key spans to be scanned).
+  optional bool ignore_misplanned_ranges = 22 [(gogoproto.nullable) = false];
```

Comment:
Remind me why its okay to change the distsql specs in a dot release?   Presumably protobuf handles new fields okay and this will just always be false when a new release binary unpacks a old release TableReaderSpec?